test/unittests/api_test.cpp

```diff
@@ -47,7 +47,7 @@ TEST(api, execution_result_value)
     const ExecutionResult result = Value{1234};
```

Comment:
```suggestion
    const ExecutionResult result = Value{1234_u32};
```

Reply:
The way `Value` constuctor behaves with `int` should be checked by `Value` unit tests.